libs/ledgerjs/script/crypto-assets-importer/importers/ethereum-plugins.js

```diff
@@ -6,7 +6,7 @@ const { readFileJSON } = require("../utils");
 const mapObject = (obj, fn) => Object.fromEntries(Object.entries(obj).map(fn));
 
 module.exports = {
-  paths: ["dapps/ethereum", "dapps/bsc", "dapps/polygon"],
+  paths: ["dapps/ethereum", "dapps/bsc", "dapps/polygon", "dapps/ethereum_ropsten", "dapps_goerli"],
```

Comment:
I don't understand how dapps_goerli can work. I don't see in https://github.com/LedgerHQ/crypto-assets/tree/main/assets

Reply:
thank you @gre I have fixed the path